CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [0.1.0] - 2019-xx-xx
```

Comment:
Planning to start the spinoff modules as 0.1.0 for basic versions that perform the same functions as the pysat versions, then increment as they become more mature.

Reply:
Fair